test/e2e/operator/garden/create_delete.go

```diff
@@ -142,7 +142,7 @@ var _ = Describe("Garden Tests", Label("Garden", "default"), func() {
 				kubernetes.WithClientOptions(client.Options{Scheme: operatorclient.VirtualScheme}),
 			)
 			g.Expect(err).NotTo(HaveOccurred())
-			g.Expect(virtualClusterClient.Client().List(ctx, &corev1.NamespaceList{})).To(Succeed())
+			g.Expect(virtualClusterClient.Client().List(ctx, &corev1.NamespaceList{})).To(Succeed()) // This
```

Comment:
?

Reply:
Oops, left over from what to check in details during the implementation. Will remove it. 
Thanks for noticing it!